### DISCUSSION TITLE
Carrier name missing in SellerCentral

### DISCUSSION THREAD

**Discussion by rokka** [abuzuhri/Amazon-SP-API-CSharp#463] [General]
Hey,

scince moving to SP-API we are expreriencing a string problem that the carrier name is not shown up in SellerCentral.

![image](https://user-images.githubusercontent.com/533290/209141278-23a6ebe4-ce15-4171-8699-622d0d3e103e.png)

Does someone else got this problem?

The XML we send looks like:

```
<?xml version="1.0" encoding="utf-16"?>
<AmazonEnvelope xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <Header>
    <DocumentVersion>1.02</DocumentVersion>
    <MerchantIdentifier>xxx</MerchantIdentifier>
  </Header>
  <MessageType>OrderFulfillment</MessageType>
  <Message>
    <MessageID>1</MessageID>
    <OperationType>Update</OperationType>
    <OrderFulfillment>
      <AmazonOrderID>xxx</AmazonOrderID>
      <FulfillmentDate>2022-12-21T11:49:48.000Z</FulfillmentDate>
      <FulfillmentData>
        <CarrierCode>DHL</CarrierCode>
        <CarrierName />
        <ShippingMethod>Post DHL o.B.</ShippingMethod>
        <ShipperTrackingNumber>xxx</ShipperTrackingNumber>
      </FulfillmentData>
    </OrderFulfillment>
  </Message>
</AmazonEnvelope>
```

And there is no error in the Amazon response.

**Comment by ThomasL1973** [comment]
In The XML you send the CarrierName Tag is empty.  What do you expect to be shown?

in deiner gesendeten XML ist der CarrierName Knoten leer. Was erwartest du was er dann anzeigt?

**Comment by rokka** [comment]
I would expect the carrier name to be determined from the CarrierCode. That was also the case at MWS and also with some of our customers.

But I will try to remove the CarrierName completly.

**Comment by rokka** [comment]
Hey,

removing the CarrierName Tag helped.

Thanks
